.eslintrc

```diff
@@ -138,6 +138,12 @@
         "react/prop-types": ["error", { "ignore": ["navigation"] }]
       }
     },
+    {
+      "files": ["packages/ui-components/**"],
```

Comment:
unlike other packages, ui components actually does need to have dev dependencies so that packages such as react are not included in the ui-components bundle.